src/service/service.py

```diff
@@ -404,4 +570,12 @@ async def health_check():
     return health_status
 
 
+@app.get("/agui/")
+async def agui_root():
+    with open("web-agui/index.html", encoding="utf-8") as file:
```

Comment:
Just noting that this only works currently if the service is run from the root of the repo, and does not work if the service is in docker. If we move to a CLI or a website that just be launched directly from browser (no server interaction) we could just remove this.

Reply:
This is indeed a difficult decision. If you want the agent to be a pure API service, then the web-agui function should also be separated into a rendering service like `streamlit_app`.